<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="register_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>REGISTER</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="information_Box">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>671</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>INFORMATION</string>
    </property>
    <widget class="QLineEdit" name="name_Edit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="name_label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Full name</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="date_of_birth_Edit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="age_label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Date of birth</string>
     </property>
    </widget>
    <widget class="QComboBox" name="gender_box">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>110</y>
       <width>73</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Male</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Female</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="gender_label">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>120</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
    <widget class="QLabel" name="department_label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Department</string>
     </property>
    </widget>
    <widget class="QComboBox" name="department_Box">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>73</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>AI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tester</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Developer</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="face_capture_button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>430</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TAKE FACE CAPTURE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_push_Button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
